pkg/admission/mutator/shoot.go

```diff
@@ -44,6 +40,14 @@ type shoot struct {
 	decoder runtime.Decoder
 }
 
+const (
+	overlayKey         = "overlay"
+	enabledKey         = "enabled"
+	createPodRoutesKey = "createPodRoutes"
+	calicoReleaseName  = "calico"
+	ciliumReleaseName  = "cilium"
+)
```

Comment:
Would it look cleaner if it was a :

```

Reply:
Wouldn't we have then the same problem as before, that the config won't parse if there is a new field in the nertwork config?